server/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+// const db;
+
+const app = express();
+const PORT = 3001;
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+//TO-DO: api get router
+app.get('/api/game/:id' (req, res) => {
+  const id = Number(path.basename(req.url));
+  db.getGame(id, (err, result) => {
+    if(err) {
+      res.sendStatus(500);
+      res.end();
```

Comment:
the end method isn't needed if you're using an express method that already ends the response process (like send etc.)

Reply:
thank you, will make an amendment to the code for this suggestion :)